### PR TITLE
python3Packages.opensimplex: init at 0.3

### PR DESCRIPTION
**Pull request by emilytrau**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/opensimplex/default.nix [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+  src = fetchFromGitHub {
+    owner = "lmas";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "idF5JQGnAye6z3c3YU9rsHaebB3rlHJfA8vSpjDnFeM=";
+  };
+
+  checkInputs = [ autopep8 coverage nose pycodestyle twine ];
+  checkPhase = "make test";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/opensimplex/default.nix:23]:
```suggestion
  checkPhase = ''
    nosetests tests/
  '';
```
we do not want to generate converage.

File: pkgs/development/python-modules/opensimplex/default.nix [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  version = "0.3";
+
+  src = fetchFromGitHub {
+    owner = "lmas";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "idF5JQGnAye6z3c3YU9rsHaebB3rlHJfA8vSpjDnFeM=";
+  };
+
+  checkInputs = [ autopep8 coverage nose pycodestyle twine ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/opensimplex/default.nix:22]:
```suggestion
  checkInputs = [ autopep8 nose pycodestyle twine ];
```

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 130276` at b8d04b98 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python38Packages.opensimplex</li>
    <li>python39Packages.opensimplex</li>
  </ul>
</details>

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 130276 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>2 packages built:</summary>  <ul>    <li>python38Packages.opensimplex</li>    <li>python39Packages.opensimplex</li>  </ul></details>

**Pull request merged**